### ISSUE TITLE
Back button

### ISSUE THREAD

**Issue by skoyah** [laravel/nova-issues#2077]
Would be nice to have a back button similar to the one that currently exists on lens for the resource detail page. 

This functionality is already implemented on lenses so I think this would not be a problem to add as a feature. A lot of our costumers use nova on mobile devices (tablets mainly) and having this functionality would improve a lot the UX on mobile devices.

At the time of this post, there is no other to go back from a detail view to the index page, unless you press the browsers back button, thus performing a full page refresh.

**Comment by jbrooksuk** [comment]
@davidhemphill will know why there is a back button on the lenses page, but this is essentially breadcrumbs and not something we'll be implementing. However, once https://github.com/laravel/nova/pull/679 is merged, you'll be able to override the headers and customise this as you wish.

**Comment by skoyah** [comment]
Thanks for you feedback @jbrooksuk  Just submitted a PR for this feature and only now I saw your comment! 😄 

Anyway, this was just a simple feature to be consistent with the lenses functionality. IMHO breadcrumbs are useful for really deep navigation. This case was more just of an easy and simple way to go back to the index page.

Thanks for your time anyway! ;)

**Comment by jbrooksuk** [comment]
As I say, David will know why the lenses have this functionality so we will leave it with him :)
